Healthy/Classes/AppDelegate.swift

```diff
@@ -30,4 +31,29 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
 }
+
+// MARK: - Authentication Handler
+
+extension AppDelegate {
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        return GIDSignIn.sharedInstance.handle(url)
+    }
+}
+
+// MARK: - Helper Methods
+
+private extension AppDelegate {
+    func gidSignInConfiguration() {
+        let config = GIDConfiguration(clientID: Constatns.clientID)
+        GIDSignIn.sharedInstance.configuration = config
+    }
+}
+
+// MARK: - Constants
+
+private extension AppDelegate {
+    enum Constatns {
+        static let clientID = "500241227951-jfe9f5o8li3l753c2146hqfru8aaa7o5.apps.googleusercontent.com"
+    }
```

Comment:
This could be moved to a separate constraints. 

Let's rename it to 
```suggestion
    enum Constatns {
        static let googleClientId = "500241227951-jfe9f5o8li3l753c2146hqfru8aaa7o5.apps.googleusercontent.com"
    }
```

Reply:
Please add a todo to update the app client id